scripts/build-preload.js

```diff
@@ -17,16 +17,12 @@ function ensureOnlyRequiresElectron(code) {
 
 const ipcModule = readFile('../src/common/ipc.ts').replace('export', '');
 
-function transpileFile(input, output, esm) {
-  const module = esm ? ts.ModuleKind.ES2020 : ts.ModuleKind.CommonJS;
-
+function transpileFile(input, output) {
   // Because we're not using a proper bundler, we need to replace the import for ipc with inlined code
   const file = readFile(input).replace("import { IPCChannel } from '../common/ipc'", ipcModule);
-  const code = ts.transpile(file, { removeComments: true, module });
+  const code = ts.transpile(file, { removeComments: true, module: ts.ModuleKind.CommonJS });

```

Comment:
I was gonna say if this works on all versions or not, but I guess we can check with the e2e tests 🎉